src/editortoolkit.cpp

```diff
@@ -229,6 +229,10 @@ bool EditorToolkit::Drag(std::string elementId, int x, int y, bool isChain)
     if (!element) {
         element = m_doc->FindChildByUuid(elementId);
     }
+    if (!element) {
```

Comment:
Note: much of what was in `src/editortoolkit.cpp` is now in `src/editortoolkit_neume.cpp`. The toolkit was split into three different toolkits (one for CMN, mensural, and neume) during the PR.